### PR TITLE
completed four column layout-Bo Mercado

### PR DESCRIPTION
**Pull request by rbomercado**
image

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Bo! The icons below the phone images are too large as compared to the example, please revise and submit a new screenshot with your revision.

**Comment by rbomercado** [comment]
image

File: css-four-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (codingcodymiller) [inline css-four-column-layout/index.html:59]: Please remove all this empty space.

File: css-four-column-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    </div>
+
+
+
+
+
+
+
+
+
```
Comment (codingcodymiller) [inline css-four-column-layout/index.html:69]: Please remove all this empty space.

File: css-four-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  margin: auto;
+  width: 800px;
+  background-color: rgb(247 248 250);
+}
+
+.row {
+  display: flex;
+}
+
+.img1 {
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:11]: This class could use a better name.

File: css-four-column-layout/styles.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  width: 70%;
+}
+
+.img2 {
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:15]: This class could use a better name.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**